sdk/test-utils/recorder-new/test/testProxyClient.spec.ts

```diff
@@ -9,26 +9,27 @@ import {
 } from "@azure/core-rest-pipeline";
 import { env } from "@azure-tools/test-recorder";
 import { expect } from "chai";
-import { TestProxyHttpClient } from "../src";
+import { RecorderClient } from "../src";
 import { RecorderError, RecordingStateManager } from "../src/utils/utils";
 
 const testRedirectedRequest = (
-  client: TestProxyHttpClient,
+  client: RecorderClient,
   makeRequest: () => PipelineRequest,
+  recordingId: string | undefined,
```

Comment:
move it to be the last parameter and optional?

Reply:
doesn't exist anymore. 🙂